### PR TITLE
Small documentation update.

### PR DESCRIPTION
**Pull request by driessamyn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ belong in this repository.
 
 To modify the version number edit the root build.gradle.
 
-The version number should track the Corda version number it is built for. Eg; Corda `4.0` should be built as `4.0.x`.
+Until `v4.0`, the version number was tracking the Corda version number it was built for. Eg; Corda `4.0` was built as `4.0.x`.
+This was broken from `v4.5` onwards (Corda 4.5 and plugins 5.0). The major version number will change when there is a breaking change,
```
Comment (chrisr3) [inline README.md:15]: Actually, l've double-checked and this should be `v4.3 onwards` - Corda 4.3 used Corda Gradle plugins 5.0.4. (I'm not sure what happened to Cordas 4.1 and 4.2... :scream:)

**Review by chrisr3 — CHANGES_REQUESTED** [review]
Small historical correction.

File: README.md (hunk 1 above)
Comment (driessamyn) [reply README.md]: Updated.
I learned yesterday, there wasn't an 4.2 OS release, only ENT. And 4.1 was definitely using Gradle v4.x (we're trying to migrate the build to Jenkins).

**Review by chrisr3 — APPROVED** [review]
LGTM :man_teacher:

**Pull request merged**